.eslintrc.js

```diff
@@ -1,10 +1,3 @@
 module.exports = {
-  root: true,
-  parserOptions: {
-    parser: 'babel-eslint',
-    sourceType: 'module'
-  },
-  extends: [
-    '@nuxtjs'
-  ]
+  extends: ['@siroc']
```

Comment:
I don't see the reason using a different eslint preset. We may update nuxtjs one if changes required 😊